cypress/pages/workflow.ts

```diff
@@ -3,9 +3,46 @@ import { BasePage } from "./base";
 export class WorkflowPage extends BasePage {
 	url = '/workflow/new';
 	getters = {
-		workflowNameInput: () => cy.getByTestId('workflow-name-input', { timeout: 5000 }).then($el => cy.wrap($el.find('input'))),
+		// workflowNameInput: () => cy.getByTestId('workflow-name-input', { timeout: 5000 }).then($el => cy.wrap($el.find('input'))),
 		workflowImportInput: () => cy.getByTestId('workflow-import-input'),
 		workflowTags: () => cy.getByTestId('workflow-tags'),
-		saveButton: () => cy.getByTestId('save-button'),
+		saveButton: () => cy.getByTestId('wf-save-button').children().first(),
+		activatorSwitch: () => cy.getByTestId('wf-activate-switch'),
+		workflowMenu: () => cy.getByTestId('workflow-menu'),
+		firstStepButton: () => cy.getByTestId('canvas-add-button'),
+		triggerNodeItem: (name: string) => cy.getByTestId(name),
+		workflowNameInput: () => cy.getByTestId('workflow-name-input'),
+	};
+	actions = {
+		visit: () => {
+			cy.visit(this.url);
+			cy.get('[data-test-id=node-view-loader]', { timeout: 5000 }).should('not.exist');
```

Comment:
We have a helper to get elements by the IDs: `getByTestId`. And you can still pass the `timeout` argument if needed as it passes down to cy.get

Reply:
Oh, Nice